configuration/messageproperties/messages_en.properties

```diff
@@ -3519,3 +3519,10 @@ pih.app.hiv.viralLoadHistory.lld=Lower limit of detection (copies/ml)
 
 emr.dataClerk.required=Data Clerk (required)
 emr.psychologue.required=Psychologue (required)
+
+pihcore.mentalhealth.if.other.service=If other service?
+pih.app.mental.health.emergencyRoom=Emergency room
+pih.app.mental.health.surgery=Surgery
+pih.app.mental.health.internal=Internal Medicine
+pih.app.mental.health.other=Other
```

Comment:
You don't need new messages when they exist.  Please re-use these:

- pihcore.other
- pihcore.location.ed
- pihcore.location.surgery
- pihcore.location.intMed